openfisca_us/parameters/usda/usda_disabled.yaml

```diff
@@ -0,0 +1,18 @@
+description: USDA Definitions for Disabled Individuals
+
+2021-01-01:
+  - federal_disability_or_blindess_receipt
+  - state_disability_or_blindess_receipt
+  - gov_agency_permanent_disability
+  - railroad_retirement_and_medicare
+  - ssi_disabled
+  - permanently_disabled_veteran
+  - surviving_spouse_of_disabled_veteran
+  - surviving_child_of_disabled_veteran
+metadata:
+  unit: program participation
```

Comment:
move into main metadata

@nikhilwoodruff is there another unit appropriate for this list?

Reply:
moved, let me know if there is a better unit